GSoC25/entity-linking-master/hybrid_linking/dbpedia_sparql.py

```
@@ -0,0 +1,31 @@
+    '''
+    sparql.setQuery(query)
+    sparql.setReturnFormat(JSON)
+    try:
+        results = sparql.query().convert()
+        candidates = []
+        for result in results["results"]["bindings"]:
+            uri = result["uri"]["value"]
+            label = result["label"]["value"]
+            candidates.append((uri, label))
+        return candidates
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)